androidApp/src/main/java/com/EENX15_22_17/digital_journal/android/ui/current/currentpatients/PatientsData.kt

```diff
@@ -0,0 +1,12 @@
+package com.EENX15_22_17.digital_journal.android.ui.current.currentpatients
+
```

Comment:
Same here, dont like the renamed file here, it was better before with "CurrentPatientsData"